### PR TITLE
Feature run intellij instructions

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,17 +21,45 @@ This repository provides end to end test for the TDR application.
...
+
+### Run the tests from Intellij
+
+1. Go to the Feature or Scenario to run
+2. Click on the double green arrows in the margin next to the Feature/Scenario
+3. Select the "Create Run Configuration" option from the menu
+4. In the dialog box fill in the following information:
+  * **Glue**: `"classpath:steps/"` (note include the double quotes)
+  * **VM options**: `-Dkeycloak.user.admin.secret=[local tdr user admin client secret] -Dkeycloak.backendchecks.secret=[backend checks for stage that tests are being run against]`
+  * **Environment Variables**: `DRIVER_LOCATION=[location of driver];INTG_AWS_ACCOUNT_NUMBER=[account number for the integration environment]`
```
Comment (suzannehamilton) [inline README.md:59]: For Mac users, does `FIREFOX_BINARY_LOCATION` have to be set here as well?

Comment (TomJKing) [reply README.md:59]: Yeah good point

**Review by suzannehamilton — COMMENTED** [review]
The driver stuff all looks correct to me because I've had to do something similar.

I haven't approved this because I haven't tried the IntelliJ glue steps yet. I'll see if I can find time soon, but it might be easier to see if Nigel, Loki or Pia has time to test them out if they're working with the e2e tests anyway?

**Comment by TomJKing** [comment]
@suzannehamilton Have got both the command line and intellij running correctly on Rooha's Mac. Added an additional change to allow the user to set the base url as an environment variable to make it easier to test against their locally running transfer frontend (`localhost:9000`)

**Review by suzannehamilton — APPROVED** [review]
Sounds good, thanks.

I've made a card about making all of these config values consistent: https://national-archives.atlassian.net/browse/TDR-909 I think they way you've done it with an overridden environment variable is the way to go.

**Pull request merged**